Include test plan for kyc-fill-in operation

ToshiWakayama-KDDI opened the pull request:
#### What type of PR is this?

* tests

#### What this PR does / why we need it:

Include kyc-fill-in test plan to be added to the Release r1.2.
This test plan covers only the base case.

#### Which issue(s) this PR fixes:

Fixes #141

#### Special notes for reviewers:



#### Changelog input

```
Include KYC-Fill-in test plan

```

#### Additional documentation

ToshiWakayama-KDDI committed:
include fill-in test plan

ToshiWakayama-KDDI commented:
Hi @fernandopradocabrillo, @GillesInnov35 , all, 

I have just open an Issue and a PR for KYC Fill-in Test file in the similar way as the Match.  It includes only one sunny case, though...  Please review if you have time.

Best regards,
Toshi
KDDI

fernandopradocabrillo commented on code/Test_definitions/kyc-fill-in.feature:
```diff
@@ -0,0 +1,38 @@
+  Scenario: Validation for success fill-in request and response scenario
+
+    Given the request body does not have any property
+    And the resource "/kyc-fill-in/v0.2rc1"
```
```suggestion
    And the resource "/kyc-fill-in/v0.2"
```

GillesInnov35 commented:
thanks @ToshiWakayama-KDDI , we could also include the error cases described by Fernando in the PR **_test plan for kyc-match-in operation_**
BR
Gilles

GillesInnov35 committed:
Update kyc-fill-in.feature

Add error cases for Test definition plan.
Gilles

ToshiWakayama-KDDI commented:
Hi @GillesInnov35 ,  cc @fernandopradocabrillo ,

Thank you so much for adding test cases in the Fill-in test file.

Just one comment and proposal.  Probably we do not need the scenario 400.01 invalid phoneNumber, if I understand correctly, because the Fill-in request body does not have any parameter/property, so it is not possible to 'set to a valid property $.phoneNumber set to <invalid_phone_number>' (line 24).

If you agree, I will remove this scenario from the Fill-in feature.

Other than the above, LGTM.

BR
Toshi

ToshiWakayama-KDDI commented:
Hi @GillesInnov35 , cc @fernandopradocabrillo ,

I have found a problem.  The test case you proposed, 'phone_number_does_not_exist_in_access_token', uses code 403 / KNOW_YOUR_CUSTOMER.INVALID_TOKEN_CONTEXT.  However, KYC Fill-in defintion does not have this error code.  Code 401 UNAUTHENTICATED could be used.  But, if we use 401, it would be different from KYC Match's.  So, my proposal is to remove this test case at present.

BR
Toshi

ToshiWakayama-KDDI committed:
Update kyc-fill-in.feature

Removed Invalid_phoneNumber
Removed phone_number_does_not_exist_in_access_token

ToshiWakayama-KDDI commented:
Hi @GillesInnov35 ,

Could you review this PR, please?

Thanks,
Toshi

GillesInnov35 reviewed (approved):
OK Thanks Toshi

ToshiWakayama-KDDI commented:
I am merging this PR, as we agreed during today's meeting.  Thanks.
